Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>I'd say more like...</p><p> </p><p>We sample using Metropolis-Hastings, using the Exchange Rule to propose new states. The exchange rule swaps the spin of two neighbouring sites, therefore it keeps the magnetisation fixed (in this case, 0).</p><p> </p><p>uzing -&gt; using</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yeah, that is a good rewriting of that sentence</p>